docs/how_to_use/how_to_build.md

```diff
@@ -266,3 +266,7 @@ unzip -o bazel-bin/docs/doxygen/tachyon_api_docs.zip &&
 google-chrome bazel-bin/docs/doxygen/html/index.html
 # generate HTML files and open on Chrome browser.
 ```
+
+### Build without assembly-optimized prime field
+
+You may encounter an illegal instruction error when running unit tests. Although the exact cause is not yet known, thanks to @zkbitcoin, we discovered that this issue is related to [ffiasm](https://github.com/iden3/ffiasm). Temporarily disabling assembly-optimized prime field can resolve this problem.
```

Comment:
Let's add more detail to run with `--//:has_asm_prime_field=false` with example command or by adding description. People won't know how to temporarily disable assembly optimized prime fields unless the open the code of BUILD.bazel.